src/main/java/gregicadditions/recipes/GARecipeAddition.java

```diff
@@ -551,6 +553,16 @@ public static void init2() {
 		ModHandler.addShapedRecipe("2x2_schematic", GAMetaItems.SCHEMATIC_2X2.getStackForm(), " d ", " S ", "   ", 'S', GAMetaItems.SCHEMATIC.getStackForm());
 		ModHandler.addShapedRecipe("dust_schematic", GAMetaItems.SCHEMATIC_DUST.getStackForm(), "   ", " S ", "  d", 'S', GAMetaItems.SCHEMATIC.getStackForm());
 
+
+		//Fluid Regulators
+		for(int i = 5; i <= 7; i++) {
```

Comment:
There's a lot of numbers without context being used here but I'm not sure exactly what we could do to make that more clear. For example I'm guessing that 30 is the base recipe voltage, and 32 is... the volts at LV? I get that you're trying to make them all the equivalent partial amp of power cost at their tier and take 5s to craft.

I'm also wondering if there's not a named ordinal for tiers or something we could be using instead of 5 and 7.